static/app/views/replays/replayTable/tableCell.tsx

```diff
@@ -369,7 +371,7 @@ export function ReplayCell({
   );
 
   return (
-    <Item isWidget={isWidget} isReplayCell>
+    <Item isWidget={isWidget} isReplayCell className={className}>
```

Comment:
is the `className` used anywhere? (might have missed it)

Reply:
@michellewzhang it's used for styling the component